content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -23,7 +23,7 @@ There are some steps to do so below. On Unix run as your Gogs user:
 * Download the file matching your platform from the [downloads page](https://dl.gitea.io/gitea)
 * Put the binary at the desired install location
 * Copy `gogs/custom/conf/app.ini` to `gitea/custom/conf/app.ini`
-* If you have custom `templates, public` in `gogs/custom/` copy them to `gitea/custom/`.
+* If you have custom `templates, public` in `gogs/custom/` copy them to `gitea/custom/`
```

Comment:
Maybe the indication could go here directly, like adding: "(or do this in a second step and incrementally because your custom templates might not be compatible with new code anymore)"